### PR TITLE
feat: add support for constant-impedance and -current loads

### LINKED ISSUE #55: feat: add support for constant-impedance and -current loads

### PR DESCRIPTION
**Pull request by sasanjac**
Fixes #55

### CODE REVIEW

**Comment by sasanjac** [comment]
@FiedLa which SystemTypes are missing?

**Comment by FiedLa** [comment]
> @FiedLa which SystemTypes are missing?

grafik
I think these loads can be mapped as SystemTypes for the shunt loads,
constant current loads would be normal loads aswell? 
the system type REACTIVE_POWER_COMPENSATOR of ElmGenstat would be a shund load aswell? 
I dont know if i got the question right, is this the answer to your question?

**Comment by sasanjac** [comment]
do you have the english terms?

**Comment by FiedLa** [comment]
> do you have the english terms?

grafik

File: psdm/topology/load_model.py [hunk 1]
```diff
@@ -13,8 +13,8 @@
 class LoadModel(Base):
     """Load Representation Based on Polynomial Model.
 
-    Load = Load0*(c_p*(U/U_0)^exp_p + c_i*(U/U_0)^exp_i + (c_z)*(U/U_0)^exp_z)
-    c_z = 1 - c_p - c_i
+    load = load_0*(c_p*(U/U_0)^exp_p + c_i*(U/U_0)^exp_i + (c_z)*(U/U_0)^exp_z)
+    c_z = 1 - c_o - c_i
```
Comment (SebastianDD) [inline psdm/topology/load_model.py:17]:
```suggestion
    c_z = 1 - c_p - c_i
```

**Pull request merged**